js/plugin/src/post-edit.js

```diff
@@ -166,6 +166,15 @@ function WPSimplechartApp( $ ) {
 	 */
 	function saveChart( data ) {
 		Object.keys( data ).forEach( function( key ) {
+			// If subtitle is set, rip it out and save it separately
+			if ('chartMetadata' === key) {
```

Comment:
I think it makes sense to move the `chartOptions.height` logic to this function since it's a similar case, and rename `handleSpecialCases()` to `setPostTitleField()`